src/report/graphs.py

```diff
@@ -52,20 +59,23 @@ def setup():
     fig_width = fig_width_pt * inches_per_pt  # width in inches
     fig_height = fig_width * golden_mean  # height in inches
     fig_size = [fig_width, fig_height]
-    params = {'backend': 'pdf',
-              # 'font.family': 'sans-serif',
-              # 'font.sans-serif': ['Avenir Next'],
-              'axes.labelsize': 10,
-              'legend.fontsize': 8,
-              'xtick.labelsize': 8,
-              'ytick.labelsize': 8,
-              'font.size': 10,
-              'text.usetex': False,
-              'figure.figsize': fig_size}
+    params = {
+        "backend": "pdf",
+        # 'font.family': 'sans-serif',
+        # 'font.sans-serif': ['Avenir Next'],
```

Comment:
Do we need to keep these commented-out lines around?

Reply:
See commit 5f63772.